packages/core/content-type-builder/admin/src/translations/en.json

```
@@ -60,7 +60,7 @@
   "error.contentTypeName.reserved-name": "This name cannot be used in your project as it might break other functionalities",
   "error.validation.enum-duplicate": "Duplicate values are not allowed",
   "error.validation.enum-empty-string": "Empty strings are not allowed",
-  "error.validation.enum-number": "Values cannot start with a number",
+  "error.validation.enum-regex": "At least one value contains invalid characters. Values should start with an alphabetical character preceeding the first occurence of a number.",
```

They don't contain invalid characters, it's the order that is incorrect.

```suggestion
"error.validation.enum-regex": "At least one value is invalid. Values should have at least one alphabetical character preceeding the first occurence of a number."
```
?

---

I like that one! Thanks!